packages/shared/src/components/post/PostModalActions.tsx

```diff
@@ -89,22 +91,24 @@ export function PostModalActions({
 
   return (
     <Container {...props} className={classNames('gap-2', className)}>
-      <SimpleTooltip
-        placement="bottom"
-        content="Read article"
-        disabled={!inlineActions}
-      >
-        <Button
-          className={inlineActions ? 'btn-tertiary' : 'btn-secondary'}
-          tag="a"
-          href={post.permalink}
-          target="_blank"
-          icon={<OpenLinkIcon />}
-          onClick={onReadArticle}
+      {onReadArticle && (
+        <SimpleTooltip
+          placement="bottom"
+          content="Read article"
+          disabled={!inlineActions}
         >
-          {!inlineActions && 'Read article'}
-        </Button>
-      </SimpleTooltip>
+          <Button
+            className={inlineActions ? 'btn-tertiary' : 'btn-secondary'}
+            tag="a"
+            href={post.permalink}
```

Comment:
```suggestion
            href={post.sharedPost.permalink}
```

Reply:
Good catch! Let me add a fallback to the original post permalink as this is used in regular post modal as well.